elements/urth-core-dataframe/urth-core-dataframe.html

```diff
@@ -125,6 +125,15 @@
             },
 
             /**
+             * Column type metadata for the DataFrame
+             */
+             column_types: {
```

Comment:
convention is camelCase for property names.  Polymer will then automagically translate attributes like camel-case to/from camelCase